IdokladSdk.IntegrationTests/Tests/Clients/ReceivedDocumentPayment/ReceivedDocumentPaymentTests.cs

```diff
@@ -91,6 +93,49 @@ public async Task List_WithFilter_PaymentOptionId_ReturnsListAsync()
             Assert.That(deleted, Is.True);
         }
 
+        [Test]
+        public async Task BatchPostAsync_SuccessfullyPosted()
+        {
+            // Arrange
+            var unpaidInvoiceId = await PostReceivedInvoiceAsync();
+
+            // Act
+            var defaultPayment = await _receivedDocumentPaymentClient.DefaultAsync(unpaidInvoiceId).AssertResult();
+            var batch = new List<ReceivedDocumentPaymentPostModel> { defaultPayment };
+            var postedPayments = await _receivedDocumentPaymentClient.PostAsync(batch).AssertResult();
+            var postedPayment = postedPayments.First();
+            var retrievedPayment = await _receivedDocumentPaymentClient.Detail(postedPayment.Id).GetAsync().AssertResult();
+            var deleted = await _receivedDocumentPaymentClient.DeleteAsync(retrievedPayment.Id).AssertResult();
+
+            // Assert
+            Assert.That(defaultPayment.InvoiceId, Is.EqualTo(unpaidInvoiceId));
+            Assert.That(postedPayment.InvoiceId, Is.EqualTo(unpaidInvoiceId));
+            Assert.That(retrievedPayment.InvoiceId, Is.EqualTo(unpaidInvoiceId));
+            Assert.That(postedPayment.Id, Is.EqualTo(retrievedPayment.Id));
+            Assert.That(deleted, Is.True);
+        }
+
+        [Test]
+        public async Task BatchDeleteAsync_SuccessfullyPosted()
+        {
+            // Arrange
+            var unpaidInvoiceId = await PostReceivedInvoiceAsync();
+            var defaultPayment = await _receivedDocumentPaymentClient.DefaultAsync(unpaidInvoiceId).AssertResult();
+            var batch = new List<ReceivedDocumentPaymentPostModel> { defaultPayment };
+            var postedPayments = await _receivedDocumentPaymentClient.PostAsync(batch).AssertResult();
+            var postedPayment = postedPayments.First();
+            var retrievedPayment = await _receivedDocumentPaymentClient.Detail(postedPayment.Id).GetAsync().AssertResult();
+
+            // Act
+            var ids = new List<int> { postedPayment.Id };
```

Comment:
Tak jako to máš tady. Ale vlastně i to var ids = new List<int> { postedPayment.Id }; bych dal klidně to Arrange

Reply:
opraveno